osd/rosa_second_monitoring_stack_installed.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: remove second monitoring stack",
+    "description" : "Red Hat SRE has noticed a second monitoring stack installed in the ‘monitoring’ namespace. This is interfering with cluster operation and preventing Red Hat from monitoring the cluster. Remove the monitoring stack immediately. User workload monitoring is supported. See https://docs.openshift.com/rosa/monitoring/osd-understanding-the-monitoring-stack.html",
```

Comment:
"Red Hat SRE **have** noticed a second monitoring stack installed in the ‘monitoring’ namespace. This is interfering with cluster **operations** and preventing Red Hat from monitoring the cluster. **Please remove the adhoc monitoring stack and use the User Workload Monitoring instead**. See https://docs.openshift.com/rosa/monitoring/osd-understanding-the-monitoring-stack.html"